src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IAwaitExpression.cs

```diff
@@ -223,5 +223,216 @@ static void M(Task<int> t)
 
             VerifyOperationTreeAndDiagnosticsForTest<LocalDeclarationStatementSyntax>(source, expectedOperationTree, expectedDiagnostics, useLatestFrameworkReferences: true);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
```

Comment:
I don't see any tests for when the control flow is the direct target of the await. For example, `var i = await (b ? AsyncMethod1() : AsyncMethod2());`

Reply:
> I don't see any tests for when the control flow is the direct target of the await. For example, var i = await (b ? AsyncMethod1() : AsyncMethod2());

I do not think this test would bring any additional value by comparison to AwaitFlow_03

---
In reply to: [188093433](https://github.com/dotnet/roslyn/pull/26811#discussion_r188093433) [](ancestors = 188093433)